### DISCUSSION TITLE
Socket.io client hosting on webserver

### DISCUSSION THREAD

**Discussion by Notadumdum** [miguelgrinberg/python-socketio#1134] [Ideas]
How could it be possible to host a socket.io client on a webserver.. for example Apache or nginx

**Comment by miguelgrinberg** [comment]
The socket.io client has  nothing to do with apache or nginx. You can run those servers on the same box as the socket.io client, but they are completely unrelated and don't interact in any way.

**Reply by Notadumdum** [reply]
You have any idea of how I could run the client on a webserver?.. thats what I meant to say

**Reply by miguelgrinberg** [reply]
I don't, sorry. I explained that there is no relation between Nginx and Apache and a Socket.IO client.

**Reply by Notadumdum** [reply]
As Forward proxy
